tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp

```diff
@@ -67,6 +67,24 @@ geometry_msgs::msg::TransformStamped generate_stamped_transform()
   return t;
 }
 
+// Runs an equality check between a and b, allowing for the valid case that a == -b
+// https://gamedev.stackexchange.com/questions/75072/how-can-i-compare-two-quaternions-for-logical-equality/75077#75077
+bool CheckQuaternionNear(const geometry_msgs::msg::Quaternion & a, const tf2::Quaternion & b, double epsilon)
```

Comment:
```suggestion
bool CheckQuaternionNear(const geometry_msgs::msg::Quaternion & a, 
                                                   const tf2::Quaternion & b, double epsilon)
```

Addresses, `tf2_geometry_msgs.cpplint.whitespace/line_length [2] (/tmp/ws/src/geometry2/tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp:72)`